uppsala-universitet-institutionen-for-biologisk-grundutbildning-swedish.csl

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="sv-SE" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Uppsala universitet, Institutionen för biologisk grundutbildning (Swedish)</title>
+    <title-short>IBG Uppsala, SE</title-short>
+    <id>http://www.zotero.org/styles/uppsala-universitet-institutionen-for-biologisk-grundutbildning-swedish</id>
+    <link href="http://www.zotero.org/styles/uppsala-universitet-institutionen-for-biologisk-grundutbildning-swedish" rel="self"/>
+    <author>
+      <name>Per Enström</name>
+      <email>per.enstrom@ibg.uu.se</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Style for Swedish texts done at the Biology Education Centre (Institutionen för biologisk grundutbildning) at Uppsala University, Sweden.</summary>
+    <updated>2015-09-17T11:12:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="ordinal">:e</term>
+      <term name="ordinal-01">:a</term>
+      <term name="ordinal-02">:a</term>
+      <term name="ordinal-11">:e</term>
+      <term name="ordinal-12">:e</term>
+      <term name="accessed">hämtad</term>
+      <term name="page" form="short">
+        <single>s.</single>
+        <multiple>s.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol"/>
+      <et-al prefix="ljhg" suffix=".6854684" font-style="italic" text-decoration="none"/>
```

Comment:
> I tested some stuff and found out that those fields does not work for et-al.

Yes, that's correct. Since CSL 1.0.1 the prefix of "et al." is controlled by "delimiter-precedes-et-al". See http://docs.citationstyles.org/en/stable/specification.html#name and https://github.com/citation-style-language/schema/issues/101